cli/keyring_generate.go

```diff
@@ -17,12 +17,14 @@ import (
 )
 
 func MakeKeyringGenerateCommand() *cobra.Command {
-	var noEncryption bool
+	var noEncryptionKey bool
+	var noPeerKey bool
 	var cmd = &cobra.Command{
 		Use:   "generate",
 		Short: "Generate private keys",
 		Long: `Generate private keys.
 Randomly generate and store private keys in the keyring.
+By default peer and encryption keys will be generated.
```

Comment:
though: I wonder if we need to be more clear every time we mention encryption keys as there are different types of them for different context: encryption-at-rest, doc encryption, some ACP-related and so on. They grow like mushrooms these days.

Reply:
I think it is well worth being very clear here - this is a destructive operation (information is overwritten), and the keys are accessible to users via the `export` command.  I have a preference for us to list each key with it's keyring name, and a short description as to what it does.

Reply:
There's a readme section dedicated to this already.

Reply:
Can we link to it from the cli? It is very handy if the CLI is self-explanatory, I find it annoying when I have to search for CLI documentation instead of just using `--help`

Reply:
I've copied the key management section into the help for the base keyring command.